cuebot/Dockerfile

```diff
@@ -1,74 +1,24 @@
-# -----------------
-# BUILD
-# -----------------
-FROM centos:7 as build
-
-# First line after FROM should be unique to avoid --cache-from weirdness.
-RUN echo "Cuebot build stage"
-
-RUN yum install -y \
-      java-1.8.0-openjdk.x86_64 \
-      java-1.8.0-openjdk-devel.x86_64 \
-      libaio \
-      which \
-    && yum clean all
-
-WORKDIR /src
-
-ENV CUEBOT_DB_ENGINE=postgres
-
-RUN adduser builduser
-
-RUN mkdir logs
-
-COPY cuebot/build.gradle cuebot/
-COPY cuebot/gradlew cuebot/
-COPY cuebot/gradlew.bat cuebot/
-COPY cuebot/settings.gradle cuebot/
-COPY cuebot/gradle/ cuebot/gradle/
+FROM gradle:jdk11 as build
 
-COPY proto/ proto/
-COPY cuebot/src/main/resources/ cuebot/src/main/resources/
-COPY cuebot/src/test/ cuebot/src/test/
-COPY cuebot/src/main/java/ cuebot/src/main/java/
+COPY --chown=gradle:gradle ./cuebot /home/gradle/cuebot/
+COPY --chown=gradle:gradle ./proto /home/gradle/proto/
 
-RUN chmod -R 777 .
-
-# Tests must be run as a non-root user as the embedded Postgres server will not
-# work otherwise.
-RUN su -c "cd cuebot && ./gradlew build --stacktrace" builduser
-
-RUN su -c "cd cuebot && ./gradlew shadowJar --stacktrace" builduser
-
-COPY VERSION.in VERSIO[N] ./
-RUN test -e VERSION || echo "$(cat VERSION.in)-custom" | tee VERSION
-
-RUN mv cuebot/build/libs/cuebot-all.jar cuebot/build/libs/cuebot-$(cat ./VERSION)-all.jar
```

Comment:
Could you add back in a line doing something like this -- basically our build pipelines currently depends on there being a versioned JAR as an artifact of this build.

If you check our the Azure results from this PR you'll see where it's looking for the file. https://dev.azure.com/cipriano0988/OpenCue%20Testing/_build/results?buildId=182

The Azure config file lists some of those details as well. https://github.com/AcademySoftwareFoundation/OpenCue/blob/master/azure-pipelines.yml#L37